### PR TITLE
chore: add bitnami to adopters

### PR DESCRIPTION
**Pull request by hacktivist123**
Adds bitnami to adopters page

### CODE REVIEW

File: src/utils/case-studies-data.js [hunk 1]
```diff
@@ -957,6 +956,10 @@ export default {
       },
     ],
   },
+  bitnami: {
+    iconName: 'bitnami',
+    text: '<b>Bitnami</b> offers Cilium to their customers as a deployment offering for cloud native networking, observability and security',
```
Comment (xmulligan) [inline src/utils/case-studies-data.js:961]:
```suggestion
    text: '<b>Bitnami</b> offers Cilium to their customers in their application catalogue',
```

**Comment by xmulligan** [comment]
can you also add tanzu? https://github.com/cilium/cilium/pull/33835/files

**Comment by hacktivist123** [comment]
Done @xmulligan

File: src/utils/case-studies-data.js [hunk 2]
```diff
@@ -957,6 +956,14 @@ export default {
       },
     ],
   },
+  tanzu: {
+    iconName: 'tanzu',
+    text: '<b>VMWare Tanzu</b> offers Cilium as part of the Tanzu Application Catalogue',
```
Comment (xmulligan) [inline src/utils/case-studies-data.js:961]:
```suggestion
    text: '<b>VMware Tanzu</b> offers Cilium as part of the Tanzu Application Catalogue',
```

File: src/utils/case-studies-data.js [hunk 3]
```diff
@@ -957,6 +956,14 @@ export default {
       },
     ],
   },
+  tanzu: {
+    iconName: 'tanzu',
```
Comment (xmulligan) [inline src/utils/case-studies-data.js:960]: this also appear not in alphabetical order on the page

Comment (hacktivist123) [reply src/utils/case-studies-data.js:960]: Ah yes, figured it out, I used tanzu for the card name instead of VMware Tanzu

**Review by xmulligan — APPROVED** [review]

**Pull request merged**